dicom-archive/get_dicom_info.pl

```diff
@@ -190,18 +329,47 @@ sub vector_cross_product {
     return @result;
 }
 
+=pod
+
+=head3 trim($input)
+
+Remove leading and trailing spaces from the $input variable
+
+INPUT: string to remove leading and trailing spaces from
+
+RESULTS: string without leading and trailing spaces
+
+=cut
+
 sub trim {
     local($input) = @_;
     $input =~ s/^\s+//;
     $input =~ s/\s+$//;
     return $input;
 }
 
+=pod
+
+=head3 showcroft()
+
+=cut
+
 sub showcroft {
     return @croft;
 }
 
-# Routine to split a DICOM list into a perl list
+=pod
+
+=head3 split_dicom_list($dlist)
+
+Routine to split a DICOM list into a perl list
+
+INPUT: list of DICOMs
+
+RETURNS: string with the list of DICOM files
```

Comment:
The functions adds `0` to each element so I am not sure it returns a list of DICOM **files**...It looks like it's returning a list of numbers.

Reply:
You're right. Tried to make it clearer...